spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py

```diff
@@ -373,37 +369,35 @@ def _fill_send_buffer(
 
         if self._send_buffer is not None:
             self._send_buffer.clear()
-        if (self._send_buffer_times is not None and
-                len(self._send_buffer_times) != 0):
+        if self._send_buffer_times is not None and self._send_buffer_times:
```

Comment:
As previously, this cannot be done this way, as self._send_buffer_times can be a numpy array.